### PR TITLE
Converted Privacy Policy to Markdown

### PR DESCRIPTION
**Pull request by GaeaKat**

### CODE REVIEW

**Review by duckspike — APPROVED** [review]

File: content/privacy-policy.md [hunk 1]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+This Website ('Snoonet.org'), and its owners Snoonet, (collectively, 'We', 'Us', or 'Snoonet') wish to define and inform you of any and all of the instances in which your personal data and information may be saved and how it may be used by Snoonet, how it may be shared, and the level of protection we use to safeguard your personal information when using our services.
+
+---
+
+{{< toc >}}
+
+---
+
+## PERSONAL INFORMATION WE COLLECT
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 2]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+From the Snoonet Website and Support System
+
+* Email address for the purposes of account management and protection from abuse, or for responding to support requests submitted by you.
+* Internet Protocol (IP) Address* for the purposes of account management and protection from abuse. (*You may use a virtual private network or 'bouncer' service if you wish to obscure your IP address)
+* Your user ID, name, or 'nickname' for the purposes of account management, protection from abuse or responding to support requests submitted by you.
+
+---
+## USES OF PERSONAL INFORMATION COLLECTED BY US
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 3]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+* E-mail address is used to confirm account ownership, for certain limited non-marketing communications by Snoonet to users, or to respond to support requests submitted through our support system.
+* NGINX webserver logs are regularly pruned and are created by the webserver. No usernames or passwords are ever logged by the webserver.
+* Compliance with valid legal process.
+
+Snoonet does not collect or log any chat information whatsoever. However, users may log their own chats privately. Please be careful about disclosing any personal information in any chat room, including in a private message, because that information may be logged by a third party in the chat room or by the recipient of the private message which is beyond the control of Snoonet.
+
+---
+## DISCLOSURE AND USE OF PERSONAL INFORMATION
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 4]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+Snoonet may be required to provide your personal information to law enforcement in the event of a criminal investigation, criminal prosecution, or civil subpoena. Snoonet will comply with legal instruments that follow proper procedure and are sufficiently narrow in their scope as to not infringe on the civil liberties of others.
+
+---
+## DATA RETENTION
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 5]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+We retain all account information and data indefinitely unless a client requests that their information be deleted. We retain at most thirty (30) days of login activity after which the data is purged from our system. We do not maintain any logs of user chat activity. To request that your information be deleted, see the [ "How to Access and Control Your Personal Information"](#how-to-access-and-control-your-personal-information) section below.
+
+---
+## CalOPPA STATEMENT
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 6]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+ 
+The State of California requires us to post specific language related to our privacy policy. By default, Snoonet does not share your private information with any third parties aside from the disclosures already made in this privacy policy. However, if you wish to inquire into specifics and verify that Snoonet does not share the personal information of our users with third parties for direct marketing purposes, you may contact [operations@snoonet.org](mailto:operations@snoonet.org).
+
+---
+## MAINTAINING THE SECURITY OF YOUR PERSONAL INFORMATION
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 7]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+Your personal information is safeguarded by Snoonet. You may access your personal information maintained by Snoonet on our IRC network via a command or by submitting a ticket through our support system. Personal information stored by Snoonet can also be made accessible to other third-parties on the Snoonet network on an opt-in basis only. All personal information maintained by Snoonet is stored in an encrypted form on a server located in the United States.
+
+---
+## HOW WE TRANSFER PERSONAL INFORMATION COLLECTED INTERNATIONALLY
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 8]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+We collect information globally and primarily store that information in the United States.
+
+---
+## COPPA DISCLOSURE - ABOUT CHILDREN'S ONLINE PRIVACY
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 9]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+The Children's Online Privacy Protection Act (COPPA) was passed to give parents increased control over what information is collected from their children online and how such information is used. The law applies to websites and services directed to, and which knowingly collect information from, children under the age of 13. Our online services are not directed to children under the age of 13, nor is information knowingly collected from them. For additional information on COPPA protections, please see the FTC website at[ https://www.consumer.ftc.gov/articles/0031-protecting-your-childs-privacy-online](https://www.consumer.ftc.gov/articles/0031-protecting-your-childs-privacy-online).
+
+---
+## HOW TO ACCESS AND CONTROL YOUR PERSONAL INFORMATION
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

File: content/privacy-policy.md [hunk 10]
```diff
@@ -5,92 +5,101 @@ draft: false
...
+
+You may access or delete your personal information from the IRC network at any time via a command. To view information shared with Snoonet, use the &ldquo;/msg NickServ INFO &lt;nickname&gt;&rdquo; command. To delete your user account, use the "/msg NickServ DROP &lt;nickname&gt;&rdquo; which requires a secondary confirmation. You may also change the email address associated with your account via a channel command. Another means to delete the personal information maintained by Snoonet is to request that your account information be deleted by the Snoonet support team. The Snoonet support team may be reached at [https://support.snoonet.org](https://support.snoonet.org). 
+
+---
+## CHANGES TO THIS PRIVACY POLICY
+[Back to Top ](#snoonet-privacy-policy)
```
Comment (linuxdaemon) [inline content/privacy-policy.md]:
```suggestion
{{< back_to_top >}}
```

**Review by linuxdaemon — CHANGES_REQUESTED** [review]

**Review by linuxdaemon — APPROVED** [review]

**Review by n7st — APPROVED** [review]
I'm merging this now - will take a look on staging and it'll make it into the next release (just want to check the Git info works properly here).

**Pull request merged**